sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -1496,6 +1495,11 @@ class RocksDB(
     // Use RocksDBMemoryManager to calculate the memory usage accounting
     val memoryUsage = RocksDBMemoryManager.getInstanceMemoryUsage(instanceUniqueId, getMemoryUsage)
 
+    val requestedPinnedBlocksMemUsage = getDBProperty("rocksdb.block-cache-pinned-usage")
```

Comment:
do we actually need the `getDBProperty("rocksdb.block-cache-pinned-usage")` here? Because in unbounded mode, the `getDBProperty("rocksdb.block-cache-pinned-usage")` will be equal to `lruCache.getPinnedUsage()`